internal/engine/fetcher.go

```diff
@@ -80,7 +80,24 @@ func (f *Fetcher) Fetch() (*apiv1.ModuleReference, error) {
 		Digest:     "unknown",
 	}
 
-	return &mr, CopyModule(f.src, modulePath)
+	copier, err := NewCopier(f.src, modulePath)
+	if err != nil {
+		return nil, fmt.Errorf("cannot create copier, error: %w", err)
+	}
+
+	copier.AddSkipDirs(".git", "vendor")
+	if gopath, haveGoapth := os.LookupEnv("GOPATH"); haveGoapth {
+		copier.AddSkipDirs(gopath)
```

Comment:
However, this one might be a bit controversial, as env lookup kind of changes things. Making explicit config with `.timoniignore` or `.gitignore` might be more sensible, but on the other hand, I'm still not sure why files aren't read in the original location.

Reply:
Modules come from remote registries, I see no way around using some tmp dir. The situation when then module is at a local path is only during development.

Reply:
Perhaps this fetcher piece should be an interface with two implementations - local dir and OCI?
To me it looks like the use case for local dir is that it is used in development and wherever during build in CI.